ui/components/Dependencies.jsx

```diff
@@ -52,12 +52,14 @@ export default connect(
       packageSelectors.getSelectedPackageName(state.packages),
     ),
   }),
-  dispatch =>
-    bindActionCreators(
-      {
-        onDependencyClick: actions.removeDependency,
-        onAddClick: actions.addDependency,
-      },
-      dispatch,
-    ),
+  dispatch => ({
+    onAddClick: name => {
+      dispatch(uiActions.showTerminal());
```

Comment:
This stuff belongs in a saga, not in mapDispatchToProps.

Reply:
I wanted to keep things separated: by doing this it's easier to debug single actions, else, every time you trigger a ```actions.addDependency()```, it will also trigger the terminal

Reply:
Isn't that what we want?

Reply:
Yes, but I prefer avoiding side effects when doing an action... Anyway, if this is not a problem, I add them in the corresponding saga